script/1_0_2current_en_script/Ciel/Day 1/01_01_CIEL01_1_3.txt

```diff
@@ -30,24 +30,24 @@ When will human beings be able to transcend all prejudices and discrimination, a
 "Gah!?"
 "T-Tohno-kun, are you all right?!
 You just got hit on the head with something!"
-"...Yeah, I guess I'm alright."
+"...Yeah, I'll survive, just about."//Is this supposed to be sarcastic? すんでのところで助かりました
 <Page10>
 With the side of my head still smarting from the impact, I pick up the object that had fallen onto the seat.
-A metallic, shiny, palm-sized hunk of metal.
+A palm-sized lump of metal that sparkles lustrously.//メタリックに輝く手のひらサイズの金属塊。
 It's a smartphone, no matter how you look at it.
 <Page11>
 "Excuse me. Did something fly over here?"
-"...This. It was flying really fast. Does it have an app on it that lets you fly it around?"
+"...I guess you mean this. It came in at a pretty incredible angle. Do you have an app that lets you fly it around like that?"//これかな。ちょっと、すごい角度で飛んできたけど。振り回して遊ぶアプリでも入ってたの
 <Page12>
 "Are you an idiot? Thank you very much."
-The girl takes the phone as if nothing had happened and walks away to a door one row of seats away.
+The girl takes the phone as if nothing had happened and walks over to a door one row of seats away.
 <Page13>
 "......"
 "......"
 We stare at each other, nonplussed by what just happened.
 <Page14>
 Anyway, I come back to my senses.
-I clear my throat to change the atmosphere.
+I clear my throat with a cough to break the tension.//A bit liberal - こほんと咳払いをして空気を切り替える。
```

Comment:
don't think 'with a cough' is needed. I think it's pretty clear that he's coughing.